manifests/server.pp

```diff
@@ -71,6 +71,12 @@
   Boolean $ssl_weak_cert                                        = false,
   Boolean $ssl_invalid_hostnames                                = false,
   Enum['requireSSL', 'preferSSL', 'allowSSL'] $ssl_mode         = 'requireSSL',
+  Optional[Boolean] $tls                                        = undef,
+  Optional[Stdlib::Absolutepath] $tls_key                       = undef,
+  Optional[Stdlib::Absolutepath] $tls_ca                        = undef,
+  Boolean $tls_conn_without_cert                                = false,
+  Boolean $tls_invalid_hostnames                                = false,
+  Enum['requireTLS', 'preferTLS', 'allowTLS'] $tls_mode         = 'requireTLS',
```

Comment:
Although no puppet-strings exists at the moment for this project, please add the documentation for the new parameters in puppet-strings format at the top of the file. 
This will make things easier if someone wants to migrate this module to puppet-strings in the future. :smile:

Reply:
Hi @root-expert ,
I've added puppet-strings in this file for the new parameters.
If you are ok with it, I will fixup the dedicated commit with the first one.